src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer.Common/MEFProjectEngineFactories.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using Microsoft.AspNetCore.Razor.ProjectEngineHost;
+using Microsoft.CodeAnalysis.Razor;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.Common;
+
+internal static class MEFProjectEngineFactories
```

Comment:
Minor point, but I'd prefer seeing this as "Mef" rather than "MEF", similar to how this is handled in Roslyn: https://sourceroslyn.io/#q=MEF.